geometry_msgs/Point head_direction
---
bool success